distribution/resources/bin/gateway.bat

```diff
@@ -62,15 +62,43 @@ if not exist %GW_HOME%\runtime\ (
     )
 )
 
+REM Needs to identify the ballerina arguments and the last argument which is the path of executable.
+REM The path of executable should be provided as \"<path>\" to avoid ballerina when the path includes a space.
+REM BAL_ARGS variable is used to store formatted string
+set BAL_ARGS=
+:formatAndValidateCmdArgs
+if "%~1"=="-e" (
```

Comment:
Indentation need to be corrected for the label content

Reply:
resolved with https://github.com/wso2/product-microgateway/pull/725/commits/a1bb55161bc993a4b5ea01805ca049af949a85fe